src/main/java/util/FileHelper.java

```diff
@@ -0,0 +1,85 @@
+package util;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * A helper that manages file I/O.
+ */
+public class FileHelper {
```

Comment:
The static filehelper class prevents dependency injection in the future. won't this cause testing problems?